presto-native-execution/presto_cpp/main/tests/TaskManagerTest.cpp

```diff
@@ -244,8 +244,14 @@ class TaskManagerTest : public testing::Test {
 
     taskManager_ = std::make_unique<TaskManager>(
         driverExecutor_.get(), httpSrvCpuExecutor_.get(), nullptr);
+
+    auto validator =
```

Comment:
to really validate it works, you could create a mock validator that throws, and expect the the task to throw that exception.